README.md
# FIXER

A flat and vibrant WordPress tradesman’s theme for selling your services and standing out from the crowd. Fixer is big, bold and demands to be seen - giving your business the visibility it deserves.

## Fixer is an excellent choice for promoting your trade and building trust in your brand – whether you’re the butcher, the baker, or the candlestick maker.

## Colorful One-Page Layout

Show off your business on a single, scrollable page and choose colors that fit your brand.

Add as many or as few sections as you need for your services, and promote your work with fetching photography and eye-catching calls to action.

## Wall-to-Wall Mega Slider

You won’t be able to look away from the wall-to-wall mega slider, which makes full use of Fixer’s responsive design.

The built-in action buttons will quickly direct users to specific areas of your site.

## Fantastically Flat Features

Bold blocky design extends to a sortable gallery and custom page templates for your portfolio, blog, about and contact pages.

Fixer’s clear “Ein Angebot erhalten” template will have you winning over new clients in no time.

## Big Friendly Raleway, Small Friendly Raleway

Top level headings in Fixer display a super-chunky variation of the elegant sans-serif typeface Raleway.

Grab attention fast while providing a much-needed contrast to Fixer’s otherwise right-angled style.

Paragraphs use the original Raleway, making for a sleek, minimalist look with lots of character.

## Have Something To Sell?

Fixer comes packaged with MarketPress and WooCommerce elements styled for you.

Set up a slick and stylish online store in minutes without a single line of code.


## Fanfare or Faded

By default Fixer ships with some seriously eye-popping color schemes. Want to tone it down? Just nudge the global color scheme and choose your own images to match.

## CUSTOMIZATION

Take complete control over colors, fonts, styles, images, videos and content - in just a few clicks.

## RESPONSIVE

Upfront makes it easy to edit the responsive view of your site in any devices, from big tablets to tiny phones.

## ELEMENTS

Sliders, galleries, maps, social, videos, contact forms, buttons, menus, widgets, code, posts, login, tabs & more.

## Fixer Features

### A flat and vibrant business theme.

* eCommerce ready
* Built-in MarketPress styles
* Built-in WooCommerce styles
* Colorful One-Page Layout
* Wall-to-Wall Mega Slider
* Fully responsive
* Opt-in ready
* Membership ready
* Full-screen images
* Long page styling
* Bespoke contact form
* Beautiful galleries
* Custom login form
* Plugin compatible
* Social integrated
* Portfolio focused
* Upfront powered
* Drag and drop
* Front-end text editing
* SEO optimised
* Fast and efficient code


CHANGELOG.md
Change Log
============


1.4.1 - 2017-04-11
-------------------------------------------------------------------------------
- Fix: minor CoursePress compatibility update

1.4 - 2017-03-29
-------------------------------------------------------------------------------
- Add CoursePress support.

1.3 - 2016-11-23
-------------------------------------------------------------------------------
- Add MarketPress support.

1.2 - 2016-10-24
-------------------------------------------------------------------------------
- Add WooCommerce support.

1.1.1 - 2016-04-21
-------------------------------------------------------------------------------
- Fix: remove unused 3rd party scripts.
- Fix: gravatar showing on mobile.
- Fix: improve the spacing on feature image.

1.1 - 2016-03-23
-------------------------------------------------------------------------------
- Add: full Upfront v1.1 compatibility.

1.0.3 - 2016-02-29
-------------------------------------------------------------------------------
- Visual improvements

- Fix: issue with caption on gallery element.
- Fix: default preset for gallery element caption.
- Fix: adjust buttons casings.
- Fix: caps on contact element for button label.

1.0.2 - 2016-02-11
-------------------------------------------------------------------------------
- Visual improvements

- Fix: avoid conflicts with other UI elements.
- Fix: transition on resize on textarea on contact/comment elements.
- Fix: spacing on subscribe newsletter lightbox.
- Add: new preset to use as icons on pre-footer region.
- Add: links to get a quote icon & text on pre-footer.

1.0.1 - 2016-02-04
-------------------------------------------------------------------------------
- Add: compatibility popup

1.0 - 2016-01-27
-------------------------------------------------------------------------------
- New: compatibility with Upfront 1.0

0.3 - 2015-11-17
-------------------------------------------------------------------------------
- Fix: new single post layout.
- Fix: footer fixer logo cropped.
- Fix: responsive navigation update.
- Fix: Fix services link to inner service page.
- Fix: single-post featured image max-height.
- Fix: home page thumbnail height on blog section.
- Fix: gallery caption hover effect to prevent issues when "show always" is enabled.
- Fix: Reduce the limit of words on home blog posts listings.
- Fix: styling comment form and list.
- Fix: Fix content overflow when no featured image.
- Fix: responsive image variations (page and post)
- Add: responsive menu hover effect.
- Add: permalink on thumbnail on home blog posts listings.
- Add: Add ellipses for home blog listings.

0.2 - 2015-06-30
-------------------------------------------------------------------------------
- Fix: post content space on home latest posts without thumbnail
- Fix: Refactor blog lists posts title to use ellipsis
- Fix: Refactor theme settings (Colors and Typo), general styles (sidebar) and global regions (header & footer)
- Fix: Refactor blog list pagination
- Fix: Better align footer elements
- Fix: Refactor Archive Sidebar Widgets and Region (remove floated region)
- Fix: Refactor comments on single post
- Fix: Refactor mobile/responsive menu
- Add: background to home latest posts without thumbnail (after first post)
- Add: background to blog lists posts without thumbnail
- Add: some hover effects on menus, links and buttons
- Add: styles for sticky posts for blog page template
- Add: "pre-footer" global region to layouts
- Add: link to logo (header) linking to root
- Add: permalink to post date because posts without title don't have how to link (theme unit test)
- Add: comments content styles for all possible tags within theme unit test
- Add: blog lists responsive styles (tablet) for home blog list
- Add: mobile responsive styles to archive home
- Add: :focus effect on inputs for forms (general forms and specific for contacts on home)
- Add: styles for vertical menu
- Add: widgets class to use when widgets are in regions with background-colors (archive for example)
- Add: services home region as inner page (desktop & responsive)
- Add: get quote home region as inner page (desktop & responsive)
- Add: about home region as inner page (desktop & responsive)
- Add: coverage area home region as inner page (desktop & responsive)

0.1.2 - 2015-04-02
-------------------------------------------------------------------------------
- Fix: style bugs and suppress nags in post page comments.
- Fix: some bugs in single post layouts.
- Fix: global footer issue.
- Fix: make prefooter region global.

0.1.1 - 2015-02-19
-------------------------------------------------------------------------------
- Fix: menu item navigation.
- Fix: home post boxes responsive.
- Fix: various text updates.

global-regions/header.php
<?php
/* START_REGION_OUTPUT */
$region_073caf = upfront_create_region(
			array (
  'name' => 'header',
  'title' => 'Header',
  'type' => 'wide',
  'scope' => 'global',
  'container' => 'header',
  'position' => 1,
  'allow_sidebar' => true,
),
			array (
  'row' => 24,
  'background_type' => 'color',
  'nav_region' => '',
  'background_color' => 'rgba(255,255,255,0)',
  'breakpoint' => 
  (array)(array(
     'tablet' => 
    (array)(array(
       'edited' => true,
       'col' => 24,
       'row' => 2,
    )),
     'mobile' => 
    (array)(array(
       'edited' => true,
       'col' => 24,
       'row' => 11,
    )),
     'custom-1410783666947' => 
    (array)(array(
       'edited' => false,
       'col' => 24,
    )),
  )),
  'use_padding' => 0,
  'sub_regions' => 
  array (
    0 => '',
  ),
  'version' => '1.0.0',
  'bg_padding_type' => 'equal',
  'top_bg_padding_num' => 0,
  'bottom_bg_padding_num' => 0,
  'bg_padding_num' => 0,
  'region_role' => 'banner',
)
			);

$region_073caf->add_element("Uspacer", array (
  'columns' => '1',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1449647785-74945 upfront-module-spacer',
  'id' => 'module-1449647785-74945',
  'options' => 
  array (
    'type' => 'UspacerModel',
    'view_class' => 'UspacerView',
    'class' => 'c24',
    'has_settings' => 0,
    'id_slug' => 'uspacer',
    'usingNewAppearance' => true,
    'element_id' => 'spacer-object-1449647785-71011',
    'preset' => 'default',
    'current_preset' => 'default',
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 1,
  'hide' => 0,
  'toggle_hide' => 0,
  'wrapper_id' => 'wrapper-1449647785-61287',
  'new_line' => true,
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'col' => 1,
    ),
    'mobile' => 
    array (
      'col' => 1,
    ),
  ),
));

$region_073caf->add_element("Uimage", array (
  'columns' => '4',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1422416936720-1326',
  'id' => 'module-1422416936720-1326',
  'options' => 
  array (
    'src' => '{{upfront:style_url}}/images/global-regions/header/fixer-logo-111x26-8987.png',
    'srcFull' => '{{upfront:style_url}}/images/global-regions/header/fixer-logo.png',
    'srcOriginal' => '{{upfront:style_url}}/images/global-regions/header/fixer-logo.png',
    'image_title' => '',
    'alternative_text' => '',
    'include_image_caption' => false,
    'image_caption' => 'My awesome image caption',
    'caption_position' => false,
    'caption_alignment' => false,
    'caption_trigger' => 'always_show',
    'image_status' => 'ok',
    'size' => 
    (array)(array(
       'width' => 111,
       'height' => 26,
    )),
    'fullSize' => 
    (array)(array(
       'width' => 111,
       'height' => 26,
    )),
    'position' => 
    (array)(array(
       'top' => 0,
       'left' => 0,
    )),
    'marginTop' => 0,
    'element_size' => 
    (array)(array(
       'width' => 150,
       'height' => 26,
    )),
    'rotation' => 0,
    'color' => '#ufc0',
    'background' => '#000000',
    'captionBackground' => '0',
    'image_id' => '1768',
    'align' => 'left',
    'stretch' => false,
    'vstretch' => false,
    'quick_swap' => false,
    'is_locked' => true,
    'gifImage' => 0,
    'placeholder_class' => '',
    'preset' => 'default',
    'display_caption' => 'showCaption',
    'type' => 'UimageModel',
    'view_class' => 'UimageView',
    'has_settings' => 1,
    'class' => 'c24 upfront-image',
    'id_slug' => 'image',
    'when_clicked' => 'external',
    'image_link' => '{{upfront:home_url}}/',
    'link' => 
    (array)(array(
       'type' => 'homepage',
       'url' => '{{upfront:home_url}}',
       'target' => '_self',
       'display_url' => '{{upfront:home_url}}',
    )),
    'usingNewAppearance' => true,
    'element_id' => 'image-1422416936718-1443',
    'row' => 11,
    'breakpoint' => 
    (array)(array(
       'mobile' => 
      (array)(array(
         'row' => 15,
         'top_padding_use' => true,
         'top_padding_num' => 25,
         'use_padding' => 'yes',
      )),
       'tablet' => 
      (array)(array(
         'row' => 13,
         'top_padding_use' => true,
         'top_padding_num' => 20,
         'use_padding' => 'yes',
      )),
       'current_property' => 'use_padding',
    )),
    'link_target' => '_self',
    'no_padding' => '',
    'anchor' => '',
    'top_padding_use' => true,
    'top_padding_num' => 50,
    'bottom_padding_num' => '15',
    'theme_style' => '',
    'use_padding' => 'yes',
    'lock_padding' => '',
    'padding_slider' => '15',
    'padding_number' => '15',
    'left_padding_num' => '15',
    'right_padding_num' => '15',
    'current_preset' => 'default',
    'breakpoint_presets' => 
    array (
    ),
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 0,
  'hide' => 0,
  'toggle_hide' => 1,
  'wrapper_id' => 'wrapper-1422432407745-1362',
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'edited' => true,
      'col' => 4,
      'clear' => true,
      'order' => 0,
    ),
    'mobile' => 
    array (
      'edited' => true,
      'col' => 4,
      'clear' => true,
      'order' => 0,
    ),
    'custom-1410783666947' => 
    array (
      'col' => 11,
      'order' => 0,
      'clear' => true,
      'edited' => true,
    ),
  ),
  'breakpoint' => 
  array (
    'tablet' => 
    array (
      'edited' => true,
      'left' => 0,
      'col' => 4,
      'top' => 0,
      'row' => 12,
    ),
    'mobile' => 
    array (
      'edited' => true,
      'left' => 0,
      'col' => 4,
      'row' => 13,
      'top' => 0,
    ),
    'custom-1410783666947' => 
    array (
      'edited' => false,
      'left' => 7,
      'col' => 4,
      'order' => 0,
    ),
  ),
));

$region_073caf->add_element("Uspacer", array (
  'columns' => '6',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1449647785-82992 upfront-module-spacer',
  'id' => 'module-1449647785-82992',
  'options' => 
  array (
    'type' => 'UspacerModel',
    'view_class' => 'UspacerView',
    'class' => 'c24',
    'has_settings' => 0,
    'id_slug' => 'uspacer',
    'usingNewAppearance' => true,
    'element_id' => 'spacer-object-1449647785-93897',
    'preset' => 'default',
    'current_preset' => 'default',
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 1,
  'hide' => 1,
  'toggle_hide' => 0,
  'wrapper_id' => 'wrapper-1449647785-77786',
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'clear' => false,
      'order' => 1,
      'edited' => true,
      'col' => 6,
    ),
    'mobile' => 
    array (
      'col' => 6,
    ),
  ),
  'breakpoint' => 
  array (
    'tablet' => 
    array (
      'hide' => 0,
      'left' => 0,
      'col' => 6,
      'edited' => true,
    ),
  ),
));

$region_073caf->add_element("Uspacer", array (
  'columns' => '1',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1449647785-5920 upfront-module-spacer',
  'id' => 'module-1449647785-5920',
  'options' => 
  array (
    'type' => 'UspacerModel',
    'view_class' => 'UspacerView',
    'class' => 'c24',
    'has_settings' => 0,
    'id_slug' => 'uspacer',
    'usingNewAppearance' => true,
    'element_id' => 'spacer-object-1449647785-42224',
    'preset' => 'default',
    'current_preset' => 'default',
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 1,
  'hide' => 1,
  'toggle_hide' => 0,
  'wrapper_id' => 'wrapper-1449647785-34399',
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'col' => 1,
    ),
    'mobile' => 
    array (
      'clear' => false,
      'order' => 1,
      'edited' => true,
      'col' => 1,
    ),
  ),
  'breakpoint' => 
  array (
    'mobile' => 
    array (
      'hide' => 0,
      'left' => 0,
      'col' => 1,
      'edited' => true,
    ),
  ),
));

$region_073caf->add_element("Unewnavigation", array (
  'columns' => '13',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1422430251821-1124',
  'id' => 'module-1422430251821-1124',
  'options' => 
  array (
    'type' => 'UnewnavigationModel',
    'view_class' => 'UnewnavigationView',
    'class' => 'c24 upfront-navigation',
    'has_settings' => 1,
    'id_slug' => 'unewnavigation',
    'menu_items' => 
    array (
      0 => 
      (array)(array(
         'menu-item-db-id' => 189,
         'menu-item-parent-id' => '0',
         'menu-item-type' => 'custom',
         'menu-item-title' => 'SERVICES',
         'menu-item-url' => '{{upfront:home_url}}/services-inner/',
         'menu-item-object' => 'custom',
         'menu-item-object-id' => '189',
         'menu-item-target' => '',
         'menu-item-position' => 1,
         'menu-item-classes' => '',
         'link' => 
        (array)(array(
           'type' => 'entry',
           'url' => '{{upfront:home_url}}/services-inner/',
           'target' => '',
           'display_url' => '{{upfront:home_url}}/services...',
        )),
      )),
      1 => 
      (array)(array(
         'menu-item-db-id' => 190,
         'menu-item-parent-id' => '0',
         'menu-item-type' => 'custom',
         'menu-item-title' => 'BLOG',
         'menu-item-url' => '{{upfront:home_url}}/blog/',
         'menu-item-object' => 'custom',
         'menu-item-object-id' => '190',
         'menu-item-target' => '',
         'menu-item-position' => 2,
         'menu-item-classes' => '',
         'link' => 
        (array)(array(
           'type' => 'entry',
           'url' => '{{upfront:home_url}}/blog/',
           'target' => '',
           'display_url' => '{{upfront:home_url}}/blog/',
        )),
      )),
      2 => 
      (array)(array(
         'menu-item-db-id' => 191,
         'menu-item-parent-id' => '0',
         'menu-item-type' => 'custom',
         'menu-item-title' => 'EIN ANGEBOT ERHALTEN',
         'menu-item-url' => '{{upfront:home_url}}/get-quote-inner/',
         'menu-item-object' => 'custom',
         'menu-item-object-id' => '191',
         'menu-item-target' => '',
         'menu-item-position' => 3,
         'menu-item-classes' => '',
         'link' => 
        (array)(array(
           'type' => 'entry',
           'url' => '{{upfront:home_url}}/get-quote-inner/',
           'target' => '',
           'display_url' => '{{upfront:home_url}}/get-quot...',
        )),
      )),
      3 => 
      (array)(array(
         'menu-item-db-id' => 192,
         'menu-item-parent-id' => '0',
         'menu-item-type' => 'custom',
         'menu-item-title' => 'ABOUT',
         'menu-item-url' => '{{upfront:home_url}}/about-inner/',
         'menu-item-object' => 'custom',
         'menu-item-object-id' => '192',
         'menu-item-target' => '',
         'menu-item-position' => 4,
         'menu-item-classes' => '',
         'link' => 
        (array)(array(
           'type' => 'entry',
           'url' => '{{upfront:home_url}}/about-inner/',
           'target' => '',
           'display_url' => '{{upfront:home_url}}/about-in...',
        )),
      )),
      4 => 
      (array)(array(
         'menu-item-db-id' => 193,
         'menu-item-parent-id' => '0',
         'menu-item-type' => 'custom',
         'menu-item-title' => 'CONTACT US',
         'menu-item-url' => '{{upfront:home_url}}/contact-us-inner/',
         'menu-item-object' => 'custom',
         'menu-item-object-id' => '193',
         'menu-item-target' => '',
         'menu-item-position' => 5,
         'menu-item-classes' => '',
         'link' => 
        (array)(array(
           'type' => 'entry',
           'url' => '{{upfront:home_url}}/contact-us-inner/',
           'target' => '',
           'display_url' => '{{upfront:home_url}}/contact-...',
        )),
      )),
    ),
    'preset' => 'default',
    'allow_sub_nav' => 
    array (
      0 => 'no',
    ),
    'allow_new_pages' => 
    array (
    ),
    'usingNewAppearance' => true,
    'menu_style' => 'horizontal',
    'menu_alignment' => 'left',
    'element_id' => 'unewnavigation-object-1422430251820-1816',
    'initialized' => false,
    'menu_id' => false,
    'menu_slug' => 'fixer-inner-menu',
    'burger_menu' => 
    array (
    ),
    'burger_alignment' => 'left',
    'burger_over' => 'over',
    'is_floating' => 
    array (
    ),
    'anchor' => '',
    'theme_style' => '',
    'breakpoint' => 
    (array)(array(
       'desktop' => 
      (array)(array(
         'burger_alignment' => 'left',
         'burger_over' => 'over',
         'menu_style' => 'horizontal',
         'menu_alignment' => 'left',
         'is_floating' => 
        array (
        ),
         'width' => 1080,
      )),
       'tablet' => 
      (array)(array(
         'theme_style' => '_default',
         'width' => 570,
         'burger_alignment' => 'right',
         'burger_over' => 'over',
         'row' => 12,
         'menu_style' => 'triggered',
         'menu_alignment' => 'right',
         'is_floating' => 'no',
         'top_padding_use' => true,
         'top_padding_num' => 20,
         'use_padding' => 'yes',
      )),
       'mobile' => 
      (array)(array(
         'burger_alignment' => 'right',
         'burger_over' => 'over',
         'width' => 315,
         'row' => 15,
         'theme_style' => '_default',
         'menu_style' => 'triggered',
         'menu_alignment' => 'right',
         'is_floating' => 'no',
         'top_padding_use' => true,
         'top_padding_num' => 20,
         'use_padding' => 'yes',
      )),
       'current_property' => 'lock_padding',
    )),
    'row' => 19,
    'top_padding_use' => true,
    'top_padding_num' => 50,
    'bottom_padding_num' => '15',
    'use_padding' => 'yes',
    'lock_padding' => '',
    'padding_slider' => 'false',
    'padding_number' => 0,
    'left_padding_num' => '15',
    'right_padding_num' => '15',
    'current_preset' => 'default',
    'breakpoint_presets' => 
    array (
    ),
    'breakpoint_menu_id' => 
    (array)(array(
       'tablet' => 
      (array)(array(
         'menu_id' => 28,
         'menu_slug' => 'fixer-inner-menu',
      )),
       'desktop' => 
      (array)(array(
         'menu_id' => 7,
         'menu_slug' => 'fixer-inner-menu',
      )),
    )),
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 0,
  'hide' => 0,
  'toggle_hide' => 1,
  'wrapper_id' => 'wrapper-1422432419698-1772',
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'edited' => true,
      'col' => 2,
      'clear' => false,
      'order' => 1,
    ),
    'mobile' => 
    array (
      'edited' => true,
      'col' => 2,
      'clear' => false,
      'order' => 1,
    ),
    'custom-1410783666947' => 
    array (
      'col' => 16,
      'order' => 1,
      'clear' => false,
      'edited' => true,
    ),
  ),
  'breakpoint' => 
  array (
    'tablet' => 
    array (
      'edited' => true,
      'left' => 0,
      'col' => 2,
      'top' => 0,
      'row' => 11,
    ),
    'mobile' => 
    array (
      'edited' => true,
      'left' => 0,
      'col' => 2,
      'top' => 0,
      'row' => 14,
    ),
    'custom-1410783666947' => 
    array (
      'edited' => false,
      'left' => 2,
      'col' => 14,
      'order' => 0,
    ),
  ),
));

$region_073caf->add_element("PlainTxt", array (
  'columns' => '5',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1405520755970-1045',
  'id' => 'module-1405520755970-1045',
  'options' => 
  array (
    'view_class' => 'PlainTxtView',
    'id_slug' => 'plain_text',
    'usingNewAppearance' => true,
    'content' => '<p style="text-align: right;" class="">
	<strong data-redactor-tag="strong" data-verified="redactor">Ein Anruf erledigt alles</strong>
</p>
<h1 style="text-align: right;" class=""><span class="uf_font_icon" style="font-size: 27px; top: 0px;" rel="font-size: 27px; top: 0px;" data-verified="redactor" data-redactor-tag="span" data-redactor-style="font-size: 27px; top: 0px;">a </span>1300 FIXER</h1>',
    'type' => 'PlainTxtModel',
    'element_id' => 'text-object-1405520755969-1764',
    'class' => 'c24 upfront-plain_txt',
    'has_settings' => 1,
    'is_edited' => true,
    'border_style' => 'none',
    'border_width' => '0',
    'border_color' => 'rgba(0, 0, 0, 0)',
    'bg_color' => 'rgba(0, 0, 0, 0)',
    'theme_style' => '',
    'anchor' => '',
    'row' => 18,
    'breakpoint' => 
    (array)(array(
       'tablet' => 
      (array)(array(
         'row' => 14,
         'use_padding' => 'yes',
      )),
       'custom-1410783666947' => 
      (array)(array(
         'row' => 13,
      )),
       'current_property' => 'use_padding',
       'mobile' => 
      (array)(array(
         'use_padding' => 'yes',
      )),
    )),
    'top_padding_use' => true,
    'top_padding_num' => 35,
    'bottom_padding_num' => '15',
    'preset' => 'phone-number-m',
    'padding_slider' => '15',
    'use_padding' => 'yes',
    'left_padding_num' => '15',
    'right_padding_num' => '15',
    'lock_padding' => '',
    'current_preset' => 'phone-number-m',
    'breakpoint_presets' => 
    (array)(array(
       'desktop' => 
      (array)(array(
         'preset' => 'phone-number-m',
      )),
    )),
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 0,
  'hide' => 0,
  'toggle_hide' => 1,
  'wrapper_id' => 'wrapper-1432910630350-1984',
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'edited' => true,
      'col' => 4,
      'order' => 2,
      'clear' => true,
    ),
    'mobile' => 
    array (
      'edited' => true,
      'col' => 4,
      'order' => 2,
      'clear' => true,
    ),
    'custom-1410783666947' => 
    array (
      'edited' => false,
      'col' => 11,
      'order' => 0,
    ),
  ),
  'breakpoint' => 
  array (
    'tablet' => 
    array (
      'edited' => true,
      'left' => 0,
      'col' => 4,
      'top' => 0,
      'row' => 14,
      'hide' => 1,
    ),
    'mobile' => 
    array (
      'edited' => true,
      'left' => 0,
      'col' => 4,
      'top' => 0,
      'hide' => 1,
    ),
    'custom-1410783666947' => 
    array (
      'edited' => true,
      'left' => 5,
      'col' => 6,
      'order' => 0,
      'row' => 13,
      'top' => 2,
    ),
  ),
));

$region_073caf->add_element("Uspacer", array (
  'columns' => '1',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1449647785-49735 upfront-module-spacer',
  'id' => 'module-1449647785-49735',
  'options' => 
  array (
    'type' => 'UspacerModel',
    'view_class' => 'UspacerView',
    'class' => 'c24',
    'has_settings' => 0,
    'id_slug' => 'uspacer',
    'usingNewAppearance' => true,
    'element_id' => 'spacer-object-1449647785-83591',
    'preset' => 'default',
    'current_preset' => 'default',
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 1,
  'hide' => 0,
  'toggle_hide' => 0,
  'wrapper_id' => 'wrapper-1449647785-58918',
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'col' => 1,
    ),
    'mobile' => 
    array (
      'col' => 1,
    ),
  ),
));

$region_073caf->add_element("Uspacer", array (
  'columns' => '8',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1449647785-15892 upfront-module-spacer',
  'id' => 'module-1449647785-15892',
  'options' => 
  array (
    'type' => 'UspacerModel',
    'view_class' => 'UspacerView',
    'class' => 'c24',
    'has_settings' => 0,
    'id_slug' => 'uspacer',
    'usingNewAppearance' => true,
    'element_id' => 'spacer-object-1449647785-82792',
    'preset' => 'default',
    'current_preset' => 'default',
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 1,
  'hide' => 1,
  'toggle_hide' => 0,
  'wrapper_id' => 'wrapper-1449647785-27184',
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'clear' => false,
      'order' => 2,
      'edited' => true,
      'col' => 8,
    ),
    'mobile' => 
    array (
      'col' => 7,
    ),
  ),
  'breakpoint' => 
  array (
    'tablet' => 
    array (
      'hide' => 0,
      'left' => 0,
      'col' => 8,
      'edited' => true,
    ),
  ),
));

$region_073caf->add_element("Uspacer", array (
  'columns' => '3',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1449647785-67753 upfront-module-spacer',
  'id' => 'module-1449647785-67753',
  'options' => 
  array (
    'type' => 'UspacerModel',
    'view_class' => 'UspacerView',
    'class' => 'c24',
    'has_settings' => 0,
    'id_slug' => 'uspacer',
    'usingNewAppearance' => true,
    'element_id' => 'spacer-object-1449647785-30347',
    'preset' => 'default',
    'current_preset' => 'default',
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 1,
  'hide' => 1,
  'toggle_hide' => 0,
  'wrapper_id' => 'wrapper-1449647785-35699',
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'col' => 3,
    ),
    'mobile' => 
    array (
      'clear' => false,
      'order' => 2,
      'edited' => true,
      'col' => 3,
    ),
  ),
  'breakpoint' => 
  array (
    'mobile' => 
    array (
      'hide' => 0,
      'left' => 0,
      'col' => 3,
      'edited' => true,
    ),
  ),
));

$regions->add($region_073caf);

/* END_REGION_OUTPUT */
